pages/about.html

```diff
@@ -1,12 +1,47 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
-</head>
-<body>
-  
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>About Jennifer</title>
+    <link href="/styles/style.css" rel="stylesheet">
+  </head>
+
+  <body>
+    <header>
+      <h1 class="page-title"> About Jennifer </h1>
+      <nav class="nav-menu">
+        <div>
+          <a href="/pages/index.html"> Home </a>
+        </div>
+        <div>
+          <a href="/pages/portfolio.html"> Portfolio </a>
```

Comment:
Be careful using absolute paths for links/resources. They can simplify refering to files within your site, but they make the assumption that this page will be hosted at the root location of website, which isn't always true for all deployment hosts.